File: src/main/java/com/proyecto/macrofit/rutinaejercicio/model/Ejercicio.java
package com.proyecto.macrofit.rutinaejercicio.model;

import lombok.Data;

@Data
public class Ejercicio {

    private Integer idEjercicio;
    private String nombreEjercicio;
    private String descripcion;
    private String imagenEjercicio;
    private String videoEjercicio;
    private Boolean activoCatalogo;
    private String zonaMuscular;
    private String implemento;
    private String nivelDificultad;
    private String musculoObjetivo;
    
}

File: Producto/MacroFit/app/src/main/java/com/duoc/macrofit/rutinas/model/Ejercicio.kt
package com.duoc.macrofit.rutinas.model

import com.google.gson.annotations.SerializedName

data class Ejercicio(
    val idEjercicio: Int,

    val nombreEjercicio: String,

    val descripcion: String?,

    val imagenEjercicio: String?,

    val videoEjercicio: String?,

    val zonaMuscular: String?,

    val implemento: String?,

    val nivelDificultad: String?,

    val musculoObjetivo: String?,

    val activoCatalogo: Boolean?
)